lib/models/html-info.js

```diff
@@ -12,4 +13,15 @@ module.exports = class HTMLInfo {
     this.description = description;
     this.layoutsDir = layoutsDir;
   }
+
+  /**
+   * Creates a new htmlInfo object for a given layout file that extends this template.
+   *
+   * @param layout - The layout file
+   */
+  getLayoutInfo(layout) {
```

Comment:
I would actually turn this into a static method of this class instead of something from an instance. The method will need an extra parameter for the htmlInfo object then though.